bin/glite-info-update-endpoints

```diff
@@ -126,11 +151,12 @@ def get_osg_urls():
 
 def get_egi_urls(status):
 
-    egi_goc_url = "http://goc.egi.eu/gocdbpi/public/?method=get_site_list&certification_status=%s&production_status=Production" %(status,)
```

Comment:
See above comment about putting static variables like this in the code instead of reading them from a config file or environment variables.